### PR TITLE
feat: set trusted-keys and trusted-git-root and trusted-sha512-root-commit

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
Sets these to proper values so verify-commits script works

## How Has This Been Tested?
Ran verify-commits locally

## Breaking Changes
None

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: contrib/verify-commits/trusted-keys [hunk 1]
```diff
@@ -1,4 +1,3 @@
-71A3B16735405025D447E8F274810B012346C9A6
-B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B
-E777299FC265DD04793070EB944D35F9AC3DB76A
-D1DBF2C4B96F2DEBF4C16654410108112E7EA81F
+29590362EC878A81FD3C202B52527BEDABE87984
+3F5D48C9F00293CD365A3A9883592BD1400D58D9
+15191D05B5CF956FE37C95962176C4A5D01EA524
```
Comment (knst) [inline contrib/verify-commits/trusted-keys:3]: nit: sort them alphabetically

**Review by knst — COMMENTED** [review]
run for this commit:
```
$ contrib/verify-commits/verify-commits.py 
Using verify-commits data from /home/knst/projects/dash-reviews/contrib/verify-commits
There is a valid path from "HEAD" to c75fff6454c8508d60058e7331815c94c565c445 where all commits are signed!
```

added one more commit:
```bash
$ contrib/verify-commits/verify-commits.py 
Using verify-commits data from /home/knst/projects/dash-reviews/contrib/verify-commits
Tree-SHA512 did not match for commit HEAD <-----------------

$ git show --show-signature
commit 98ecf07493291c29bfb22b70aa6661fa8293bd49 (HEAD -> feat/fix-verify-commits)
gpg: Signature made Thứ ba, 12 Tháng 3 Năm 2024 23:43:21 +07
gpg:                using RSA key 15191D05B5CF956FE37C95962176C4A5D01EA524
gpg: Good signature from "Konstantin Akimov <knstqq@gmail.com>" [unknown]
gpg:                 aka "Konstantin Akimov <konstantin.akimov@dash.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 1519 1D05 B5CF 956F E37C  9596 2176 C4A5 D01E A524
Author: Konstantin Akimov <knstqq@gmail.com>
Date:   Tue Mar 12 23:43:14 2024 +0700

    nothing. signed
```

That's strange, adding any extra commits breaks the script. Is there anything I am doing wrong?

**Comment by PastaPastaPasta** [comment]
> run for this commit:
> 
> ```
> $ contrib/verify-commits/verify-commits.py 
> Using verify-commits data from /home/knst/projects/dash-reviews/contrib/verify-commits
> There is a valid path from "HEAD" to c75fff6454c8508d60058e7331815c94c565c445 where all commits are signed!
> ```
> 
> added one more commit:
> 
> ```shell
> $ contrib/verify-commits/verify-commits.py 
> Using verify-commits data from /home/knst/projects/dash-reviews/contrib/verify-commits
> Tree-SHA512 did not match for commit HEAD <-----------------
> 
> $ git show --show-signature
> commit 98ecf07493291c29bfb22b70aa6661fa8293bd49 (HEAD -> feat/fix-verify-commits)
> gpg: Signature made Thứ ba, 12 Tháng 3 Năm 2024 23:43:21 +07
> gpg:                using RSA key 15191D05B5CF956FE37C95962176C4A5D01EA524
> gpg: Good signature from "Konstantin Akimov <knstqq@gmail.com>" [unknown]
> gpg:                 aka "Konstantin Akimov <konstantin.akimov@dash.org>" [unknown]
> gpg: WARNING: This key is not certified with a trusted signature!
> gpg:          There is no indication that the signature belongs to the owner.
> Primary key fingerprint: 1519 1D05 B5CF 956F E37C  9596 2176 C4A5 D01E A524
> Author: Konstantin Akimov <knstqq@gmail.com>
> Date:   Tue Mar 12 23:43:14 2024 +0700
> 
>     nothing. signed
> ```
> 
> That's strange, adding any extra commits breaks the script. Is there anything I am doing wrong?

Yes; the commit you're adding doesn't have a SHA512 in the commit message; this script is only designed to be ran on develop / master (which has proper merge commits into it)

**Review by knst — APPROVED** [review]
ACK

I tried to add tree-sha-512 for new commit - succeed.
I did `git commit --amend` to change hash - fails as expected.

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**